### PR TITLE
8300168: Typo in AccessibleJTableHeaderEntry javadoc

### PR DESCRIPTION
**Pull request by kumarabhi006**
There was a typo error in method description of `public AccessibleJTableHeaderEntry(int c, JTableHeader p, JTable t)`. Replaced the text `AccessiblJTableHeaaderEntry` with `AccessibleJTableHeaderEntry`.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8300168](https://bugs.openjdk.org/browse/JDK-8300168): Typo in AccessibleJTableHeaderEntry javadoc

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**)
 * [Tejesh R](https://openjdk.org/census#tr) (@TejeshR13 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12024/head:pull/12024` \
`$ git checkout pull/12024`

Update a local copy of the PR: \
`$ git checkout pull/12024` \
`$ git pull https://git.openjdk.org/jdk pull/12024/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12024`

View PR using the GUI difftool: \
`$ git pr show -t 12024`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12024.diff">https://git.openjdk.org/jdk/pull/12024.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/share/classes/javax/swing/table/JTableHeader.java [hunk 1]
```diff
@@ -935,7 +935,7 @@ protected class AccessibleJTableHeaderEntry extends AccessibleContext
             private JTable table;
 
             /**
-             *  Constructs an AccessiblJTableHeaaderEntry
+             *  Constructs an AccessibleJTableHeaderEntry
```
Comment (prsadhuk) [inline src/java.desktop/share/classes/javax/swing/table/JTableHeader.java:938]: There's an extra space in front of "Constructs"..please remove that too..

Comment (kumarabhi006) [reply src/java.desktop/share/classes/javax/swing/table/JTableHeader.java]: Updated.

**Review by prsadhuk — APPROVED** [review]

**Review by TejeshR13 — APPROVED** [review]

**Pull request closed without merging**